client/app/log.go

```diff
@@ -54,7 +54,9 @@ func InitLogging(logFilename, lvl string, stdout bool, utc bool) (lm *dex.Logger
 		fmt.Fprintf(os.Stderr, "failed to create file rotator: %v\n", err)
 		os.Exit(1)
 	}
-	fmt.Println("Logging to ", logFilename)
+	if !stdout {
+		fmt.Println("Logging to", logFilename)
+	}
```

Comment:
If **not** `--stdout` then we print the message saying where the log file is.  If stdout is set, it didn't seem helpful since it replicates everything in the log here anyway.

Reply:
Can remove this, but I suspected this fmt.Println was an accidental leftover from the app package refactor.

Reply:
Gotcha. I see we weren't printing this anywhere before the app package refactor. Must've been an oversight not to restrict this.